geoportal/geoportailv3_geoportal/static-ngeo/js/apps/MainController.js

```diff
@@ -465,7 +466,7 @@ const MainController = function(
     ngeoDownload, appMvtStylingService, ngeoDebounce, geonetworkBaseUrl, appBlankLayer,
     proxyWmsUrl,
     httpsProxyUrl,
-    getvtstyleUrl, uploadvtstyleUrl, deletevtstyleUrl, vectortilesUrl) {
+    getvtpermalinkUrl, uploadvtpermalinkUrl, deletevtpermalinkUrl, vectortilesUrl) {
```

Comment:
Same question as above: Why do we need to use the  `...permalink...` entry points here instead of the existing ones?